servicemesh-install/topics/install.adoc

```diff
@@ -75,15 +71,6 @@ The 3scale Istio Adapter is deployed and configured in the custom resource file.
   kind: ControlPlane
   metadata:
```

Comment:
We should update the apiVersion and kind to match the ServiceMeshControlPlane from other examples

Reply:
I updated "apiVersion" to read "istio.openshift.com/v1alpha3" and "ControlPlane" to read "ServiceMeshControlPlane".

Reply:
apiVersion should be `maistra.io/v1`

Reply:
For that matter, everything that was prefixed with `istio.openshift.com` has been replaced with `maistra.io` for consistency and to reflect the correct project origin.

Reply:
> For that matter, everything that was prefixed with `istio.openshift.com` has been replaced with `maistra.io` for consistency and to reflect the correct project origin.

I searched all existing documentation and this was the only place we needed to make that change. Thanks @rcernich!